README.md

```diff
@@ -148,12 +148,12 @@ semantics as follows:
 There's a few ways you extract the data you want to assert on. While you can do this yourself before calling the
 assertion, these methods will add the extra context to the failure message which can be helpful.
 
-The simplest way is with `prop()`. It will take a property (or function, or a name and a lambda) and return an
+The simplest way is with `having()`. It will take a property (or function, or a name and a lambda) and return an
```

Comment:
so the downside of updating the README now is that it'll show up on the main github page even though this change hasn't landed in a release yet, so I think we'll have to wait to change it unfortunately.

Reply:
Oops, oh yeah.

Reverted in https://github.com/willowtreeapps/assertk/pull/524/commits/3af4e67fd98dd40013653663aca389c41c7b7a8a